atlas

```diff
@@ -507,18 +507,21 @@ pull_translations() {
     quiet="--quiet"
   fi
 
-  if [ "$pull_revision" ];
-  then
-    pull_revision_argument="--branch=${pull_revision}"
-  else
-    pull_revision_argument=""
-  fi
 
   # Creating a shallow clone of the repo without without pulling the files yet
-  # shellcheck disable=SC2086
-  git clone $quiet $pull_revision_argument --filter=blob:none --no-checkout --depth=1 \
-            "https://github.com/${pull_repository}.git" translations_TEMP || exit
+  git init $quiet translations_TEMP || exit
   cd translations_TEMP || exit
+  git remote add origin "https://github.com/${pull_repository}.git" || exit
+  # Requires server to allow pulling commits by direct commit hash reference,
+  # controlled by uploadpack.allowAnySHA1InWant and related configs. (GitHub
+  # does allow this, though.)
+  #
+  # The `remote.<name>.promisor` config override is needed for git 2.25 and possibly
+  # later versions (but not in 2.30 and later). In some earlier versions it might
+  # instead need to be written as `extensions.partialClone=origin` -- 2.25.1 mentions
+  # this in its output if a promisor is not set, but doesn't actually *use* that config.
```

Comment:
Some info on this inconsistency: https://lore.kernel.org/git/xmqqblp1dsih.fsf@gitster.c.googlers.com/T/